### ISSUE TITLE
We can't write in a forked project after save changes

### ISSUE THREAD

**Issue by 11joselu** [stackblitz/core#1877]
### **Description of Bug**

Embeding a code in a iframe, if the use make some changes then save it, we are not able to have access to the editor vía vm.

### **Steps to Reproduce**

Example: 

https://stackblitz.com/edit/sdk-vm-umodtq?file=index.html

1. Wait to iframe to load
2. Make some changes in README.md file
3. Save it
4. Project should be forked (blue popup should appear with text "Project forked")
5. Click on button "Write code" -> nothing happens

### **Expected Behavior**

We should write in a forked project or disable "Save" to prevent fork

### **Screenshots/Screencast**

![bug-stackblitz](https://user-images.githubusercontent.com/8685132/169259139-c4f6550f-598d-4743-9a8a-4e74eeba8563.gif)

**Additional Context/Questions**

There are any option to disable fork process (allowing user just to type without saving changes)?

**Comment by bewards** [comment]
Seeing similar issues when trying to save (fork). The dev console logs a POST 422 response code if that helps.

**Comment by fvsch** [comment]
Thanks for the detailed report @11joselu, this is definitely something we should fix.

We probably don't want forking to ever happen in an embedded project.

**Comment by fvsch** [comment]
Hi @11joselu, @bewards and onlookers!

We shipped a fix today for this issue. When saving inside an embed, we now prevent forking the project, and instead only commit the user's changes to the in-memory virtual filesystem (which lets file watchers react to changes, in a WebContainers project with a file watcher or development server running).

I tested the reproduction project at https://stackblitz.com/edit/sdk-vm-umodtq?file=index.html

Following the provided steps:

1. Wait to iframe to load -> OK
2. Make some changes in README.md file -> OK
3. Save it -> OK, project does not get forked, no blue popup appears
4. Click on "Write code" button -> OK, the file in the embed gets updated
